src/strategies/governor-alpha/index.ts

```diff
@@ -0,0 +1,57 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'snapshot-labs';
+export const version = '0.0.1';
+
+const abi = [
```

Comment:
Can you use this instead:
`function getCurrentVotes(address account) external view returns (uint96)`
same format than here: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of/index.ts#L7-L9

Reply:
@bonustrack Added that 👍 works fine